lib/python/test/setups/ESKHI/Data.py

```diff
@@ -0,0 +1,86 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2022 PIConGPU contributors
+Authors: Mika Soren Voss
+License: GPLv3+
+
+This test enables the ESKHI to be checked. It is irrelevant whether
+it is a 2D or 3D simulation. In a 3D simulation, however, attention
+must be paid to the Lorentz factor. Furthermore, the error in the
+growth rate must be taken into account in the 2D simulation (for more
+information, for example, see the bachelor thesis submitted
+by Mika Soren Voß).
+
+For information:
+    according to the documentation, the case distinction will only
+    be incorporated in a later version, which means that two case
+    s of KHI currently have to be treated differently.
+"""
+import sys
+sys.path.insert(1, "../../")
```

Comment:
why do you replace the 2nd entry of the path list?

Reply:
It's almost random. I could also have used append and simply added it afterwards. But here I'm looking for another solution anyway because flake8 is not fulfilled here.

Reply:
@MSVoss I think sth like this could possibly also work and would maybe not trigger flake8. But, no idea if this is any better. 
```python
from importlib.machinery import SourceFileLoader
testsuite = SourceFileLoader("testsuite", "../../testsuite/__init__.py").load_module()
````